data/json/items/tools.json

```
@@ -1506,6 +1506,27 @@
     "color": "cyan",
     "qualities": [ [ "SMOKE_PIPE", 1 ] ]
   },
+  {
+    "type": "TOOL",
+    "id": "crash_axe",
+    "symbol": ";",
+    "color": "light_gray",
+    "name": "crash axe",
+    "looks_like": "hatchet",
+    "//": "Same stats as a hatchet, except it has prying instead of hammering.",
+    "description": "A short lightweight emergency tool with a quarter circular blade, a short pick opposite the blade and an insulated handle.  Used on airplanes to chop down or pry walls or cabinets to gain access in case of fire.",
+    "price": 2500,
+    "price_postapoc": 5000,
+    "material": [ "steel" ],
+    "techniques": [ "WBLOCK_1" ],
+    "weight": 907,
+    "volume": 4,
```

Volumes should be specified as a string.
```suggestion
    "volume": "1 L",
```